Translation corrections (#78)
07th-mod/higurashi-console-arcs: Update/some_004.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>尾形</color>", NULL, "<color=#f5e6d3>Ogata</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_ogata1023", 256, TRUE);
	OutputLine(NULL, "「公由さんのご家庭では、受験浪人についてご理解がありますか？」",

> 			NULL, "\"Would your family at home be understanding that you can always take the entrance exam again the year after?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Kimiyoshi-san, would your family be willing to understand that you can always take the entrance exam again a year later?\"", Line_Normal);
```

	ClearMessage();

//夏美rvS04/36/310100328.「いえ、その…」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100328", 256, TRUE);
	OutputLine(NULL, "「いえ、その…」",
			NULL, "\"No, um...\"", Line_Normal);
	ClearMessage();

//rあいまいに言葉を濁す。…だけど、私の想像があからさまに表情に出て、きっと伝わってしまったのだろう。尾形先生はそっと目を伏せると、ファイルにペンを走らせた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あいまいに言葉を濁す。…だけど、私の想像があからさまに表情に出て、きっと伝わってしまったのだろう。尾形先生はそっと目を伏せると、ファイルにペンを走らせた。",

> 			NULL, "My words were vague. ...But I'm sure my thoughts showed on my face and were clearly coveyed. Ogata-sensei gently turned her eyes and ran her pen across my file.", Line_Normal);

Reviewer: ```suggestion
			NULL, "My words were vague. ...But I'm sure my thoughts were plain to see on my face. Ogata-sensei gently turned her eyes and ran her pen across my file.", Line_Normal);
```

	ClearMessage();

//rおかーさんは、私にはいい大学に行ってもらいたいと思っている。…だけど、それは現役で入学しての話だ。もしも私が受験浪人をするなんて言ったら、きっと世間体を気にして反対するだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おかーさんは、私にはいい大学に行ってもらいたいと思っている。…だけど、それは現役で入学しての話だ。もしも私が受験浪人をするなんて言ったら、きっと世間体を気にして反対するだろう。",

> 			NULL, "Mom wants me to go to a nice university. ...But she means one that I can get in on the first try. If I tell her I'd be aiming to pass the second time around, I'm sure she'll oppose it on concerns about appearances.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mom wants me to go to a nice university. ...But she means one that I can get into on the first try. If I tell her I'd be aiming to pass the second time around, I'm sure she'll be concerned about appearances and oppose it.", Line_Normal);
```
